### PR TITLE
Fix running pytests during conda build.

### PR DESCRIPTION
**Pull request by matthewwardrop**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I am currently blocked on this package being updated to R 4.0, and wanted to help out. So I spun up a docker container and poked around. The issue appears to be due to `r-cairo` not being present during the tests (specifically when testing the IPython magics which automatically set up the graphics environment), and then, with `r-cairo` now available, testing the svg arguments would fail due to no fonts being installed (thus this patch skips the svg args test).

With this patch things should start flowing again, and we can update to R 4.0!

### CODE REVIEW

**Comment by matthewwardrop** [comment]
@conda-forge-admin please rerender

**Comment by dbast** [comment]
No r-cairo available for aarch64 and pp64le.

**Comment by matthewwardrop** [comment]
Hmmm... So we can either universally disable a swathe of checks then, or selectively disable them only for arm. I'll patch this up tomorrow.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ test:
   source_files:
     - rpy2/tests
   requires:
+    - r-cairo
```
Comment (dbast) [inline recipe/meta.yaml:46]:
```suggestion
    - r-cairo  # [not (aarch64 or ppc64le)]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -58,7 +59,7 @@ test:
     - rpy2.robjects  # [not win]
     - rpy2.robjects.lib  # [not win]
   commands:
-    - pytest -s -rxs -v rpy2/tests -k "not test_display_args"  # [not win]
+    - pytest -s -rxs -v rpy2/tests -k "not test_display_args and not test_svg_plotting_args"  # [not win]
```
Comment (dbast) [inline recipe/meta.yaml:62]:
```suggestion
    - pytest -s -rxs -v rpy2/tests -k "not test_display_args and not test_svg_plotting_args"  # [not (win or aarch64 or ppc64le)]
```

Comment (dbast) [reply recipe/meta.yaml:62]: That is maybe better than disabling two many tests for all archs.

Comment (matthewwardrop) [reply recipe/meta.yaml]: Agreed. On top of these I added tests for aarch64 and ppc64le that just skip the offending tests.

**Comment by matthewwardrop** [comment]
@dbast Should be ready to merge now. All builds have passed (there seems to be a stale/duplicate build here, probably because I pushed additional changes before the previous build finished).

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
Thanks so much for taking care of this @matthewwardrop!

**Comment by matthewwardrop** [comment]
No worries @ocefpaf! Happy to help out if things go haywire again... just ping me!

**Pull request merged**